### PR TITLE
add ingress v1 e2e test

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: test/e2e/ingress.go [hunk 1]
```diff
@@ -596,7 +597,316 @@ var ____ = describe("Ingress tests custom routes", func() {
...
+
+		_, err = cs.NetworkingV1().Ingresses(ns).Get(context.TODO(), ing.Name, metav1.GetOptions{ResourceVersion: "0"})
+		Expect(err).NotTo(HaveOccurred())
+
+		// skipper http -> https redirect
+		By("Waiting for skipper route to default redirect from http to https, to see that our ingress-controller and skipper works")
+		err = waitForResponse(addr, "http", waitTime, isRedirect, true)
+		Expect(err).NotTo(HaveOccurred())
+
+		// ALB ready
```
Comment (AlexanderYastrebov) [inline test/e2e/ingress.go:666]: If we do not set LB type on ingress the type would be default LB type which could be NLB

**Comment by AlexanderYastrebov** [comment]
:+1:

File: test/e2e/ingress.go (hunk 1 above)
Comment (szuecs) [reply test/e2e/ingress.go:666]: yes true and we might want to convert the e2e tests in the future to default to NLB and use some specific tests for ALB.

**Comment by szuecs** [comment]
:+1:

**Pull request merged**